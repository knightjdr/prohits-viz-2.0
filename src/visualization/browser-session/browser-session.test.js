import React from 'react';
import { shallow } from 'enzyme';

import BrowserSession from './browser-session';

jest.mock('./browser-session__table');
jest.mock('../browser-storage/indexeddb-container');

describe('Browser stored sessions', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<BrowserSession />);
    expect(wrapper).toMatchSnapshot();
  });
});
