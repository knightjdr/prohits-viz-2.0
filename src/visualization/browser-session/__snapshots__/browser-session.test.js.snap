// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Browser stored sessions should match snapshot 1`] = `
<IndexedDBContainer
  render={[Function]}
/>
`;
